lib/src/nyxx/internal/Http.dart

```diff
@@ -296,12 +299,15 @@ class Http {
       bool beforeReady = false,
       Map<String, String> headers = const {},
       String reason}) async {
+    final Map<String, String> _headers = Map.from(this._headers)..addAll(headers);
+    if(!browser)
```

Comment:
I think web client should be also able to set audit logs

Reply:
Setting that header was not allowed due to CORS.